examples/tutorials/translations/Ukrainian/Part 01 - Основні засоби приватного глибинного навчання.ipynb

```diff
@@ -0,0 +1,960 @@
+{
```

Comment:
<p>In other languages, we've been adding a second list for "translators" below the list of "authors." </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/OpenMined/PySyft/pull/2854/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>